developer/get_examples.py

```diff
@@ -12,7 +12,9 @@
 def get_examples():
     """The current list of examples, determined from the directories on disk"""
     return [
-        os.path.basename(os.path.normpath(file)) for file in glob.glob(f"{EXAMPLES}/*")
+        os.path.basename(os.path.normpath(file))
+        for file in glob.glob(f"{EXAMPLES}/*")
+        if os.path.isdir(file) and os.path.basename(file) != "_chemiscope_sphinx_data"
```

Comment:
why is this required?

Reply:
I thought that `nox -e docs` treats all folders as the instance of the example to generate the docs.
However, with the new development, there are some generated datasets that are stored in the folder `_chemiscope_sphinx_data`.
Hence I thought to exclude this generated folder out to prevent counting it as an "example".

Reply:
what's generating data there? Ideally anything generated should be stored in a sub-directory of the main example directoty (i.e. `roy-gch/_chemiscope_sphinx_data`), and then we can gitignore this sub-directory